README.fr.md

```diff
@@ -0,0 +1,167 @@
+
+`yarn add react-big-calendar` ou `npm install --save react-big-calendar`
+
+Incluez `react-big-calendar/lib/css/react-big-calendar.css` pour avoir les styles, et vérifiez que l'élément conteneur de votre calendrier a une hauteur, sinon le calendrier ne sera pas visible. Pour fournir votre propre style personnalisé, consultez le sujet [Custom Styling](#custom-styling)
+
+## Pour commancer 
+
+- [react-big-calendar](https://github.com/arecvlohe/rbc-starter)
+- [react-big-calendar with drag and drop](https://github.com/arecvlohe/rbc-with-dnd-starter)
+- [react-big-calendar with TypeScript and React hooks bundled with Vite](https://github.com/christopher-caldwell/react-big-calendar-demo)
+
+## Exécutez des exemples localement
+
+```sh
+$ git clone git@github.com:jquense/react-big-calendar.git
+$ cd react-big-calendar
+$ yarn
+$ yarn storybook
+```
+
+- Ouvrez [localhost:3000/examples/index.html](http://localhost:3000/examples/index.html).
+
+### Localisation and formatage des dates
+
+`react-big-calendar` a quatre options pour gérer le formatage des dates et la localisation culturelle, selon votre préférence de bibliothèques DateTime. Vous pouvez utiliser un des localisateurs suivant:  [Globalize.js](https://github.com/jquery/globalize), [date-fns](https://date-fns.org/), [Day.js](https://day.js.org).
+
+Quel que soit votre choix, vous **devez** choisir un localisateur pour utiliser cette bibliothèque:
+
+#### Moment.js
+
+```js
+import { Calendar, momentLocalizer } from 'react-big-calendar'
+import moment from 'moment'
+
+const localizer = momentLocalizer(moment)
+
+const MyCalendar = (props) => (
+  <div>
+    <Calendar
+      localizer={localizer}
+      events={myEventsList}
+      startAccessor="start"
+      endAccessor="end"
+      style={{ height: 500 }}
+    />
+  </div>
+)
+```
+
+#### Globalize.js v0.1.1
+
+```js
+import { Calendar, globalizeLocalizer } from 'react-big-calendar'
+import globalize from 'globalize'
+
+const localizer = globalizeLocalizer(globalize)
+
+const MyCalendar = (props) => (
+  <div>
+    <Calendar
+      localizer={localizer}
+      events={myEventsList}
+      startAccessor="start"
+      endAccessor="end"
+      style={{ height: 500 }}
+    />
+  </div>
+)
+```
+
+#### date-fns v2
+
+```js
+import { Calendar, dateFnsLocalizer } from 'react-big-calendar'
+import format from 'date-fns/format'
+import parse from 'date-fns/parse'
+import startOfWeek from 'date-fns/startOfWeek'
+import getDay from 'date-fns/getDay'
+import enUS from 'date-fns/locale/en-US'
+
+const locales = {
+  'en-US': enUS,
+}
+
+const localizer = dateFnsLocalizer({
+  format,
+  parse,
+  startOfWeek,
+  getDay,
+  locales,
+})
+
+const MyCalendar = (props) => (
+  <div>
+    <Calendar
+      localizer={localizer}
+      events={myEventsList}
+      startAccessor="start"
+      endAccessor="end"
+      style={{ height: 500 }}
+    />
+  </div>
+)
+```
+
+#### Day.js
+
+Note that the dayjsLocalizer extends Day.js with the following plugins:
```

Comment:
You missed translating this section here

Reply:
Thank you for the reply and heads up! Fixing it right away!

Reply:
@cutterbl 
I have fixed the missing translation and force pushed the changes.
Check it out and let me know if there is anything else I should alter